README.md

```diff
@@ -10,3 +10,13 @@ Snyk helps you find, fix and monitor for known vulnerabilities in your dependenc
 ## Snyk Docker CLI Plugin
 
 This plugin provides dependency metadata for Docker images. 
+
+## Running Tests
+
+To run tests the following environment variables need to be set:
+
+`DOCKER_HUB_PRIVATE_IMAGE`
+`DOCKER_HUB_USERNAME`
+`DOCKER_HUB_PASSWORD`
+
+`DOCKER_HUB_PRIVATE_IMAGE` should refer to an image that is hosted on Docker Hub but not available publicly. During CI test this is set to `snykgoof/dockergoof:alpine`.
```

Comment:
Why do we need an env var here and not use `snykgoof/dockergoof:alpine` directly?

Reply:
2 reasons mainly: 

1. This lib is open source, so on the off chance someone wants to contribute this enables them to test locally with their own private image.
2. If we add more tests that use this down the line we'll only have to change the value in 1 place.